src/mono/wasm/Makefile

```diff
@@ -144,10 +144,10 @@ submit-tests-helix:
 
 run-debugger-tests:
 	rm -f $(TOP)/artifacts/bin/DebuggerTestSuite/x64/Debug/*log; \
-	if [ ! -z "$(TEST_FILTER)" ]; then \
-	$(DOTNET) test  $(TOP)/src/mono/wasm/debugger/DebuggerTestSuite $(MSBUILD_ARGS) "-l:trx;LogFileName=DebuggerTestsResults.xml" --results-directory $(TOP)/artifacts/log/$(CONFIG) --filter "Category!=failing&FullyQualifiedName$(TEST_FILTER)" $(TEST_ARGS); \
+	if [ ! -z "$(TEST_ARGS)" ]; then \
+	$(DOTNET) test  $(TOP)/src/mono/wasm/debugger/DebuggerTestSuite $(MSBUILD_ARGS) "-l:trx;LogFileName=DebuggerTestsResults.xml" --results-directory $(TOP)/artifacts/log/$(CONFIG) $(TEST_ARGS)"&category!=failing"; \
```

Comment:
Don't remove TEST_FILTER, it is very useful for running tests. Merge their use instead.
What is the use case for TEST_ARGS, and TEST_FILTERS being used together?

Reply:
No idea, I was convinced `TEST_FILTER` is never used. `TEST_ARGS`, to the contrary - is always set by
https://github.com/dotnet/runtime/blob/bc4c942dc042f795e736d4d0d43a6194a84d9a79/src/mono/wasm/debugger/Wasm.Debugger.Tests/Wasm.Debugger.Tests.csproj#L76
If you say that `TEST_FILTER` is used then it seems they are both set in the "useful in running tests" situations. What exactly sets `TEST_FILTER`? Is it always set or conditionally?

Reply:
What are the possible values in `TEST_FILTER`? It seems to me it duplcates the purpose of `TEST_ARGS`? I mean:
- if `TEST_FILTER` not empty then we are appending `FullyQualifiedName$(TEST_FILTER)`, supposedly `TEST_FILTER` holds the name of the test to run,
- `TEST_ARGS` does exactly the same job, it holds `--filter FullyQualifiedName~%(TestToRunName)`